steps/jira-issue-update-build/0.1.3/step.yml

```diff
@@ -0,0 +1,82 @@
+title: Update JIRA issues with build number
+summary: Update fields of JIRA issues with build number.
+description: "Update fields of JIRA issues associated with current MR/PR, with the
+  current build number.\n\n### How does it work?\n\nFirst, the step needs to know
+  tasks associated with the build, so it examines Git history of the merge that it
+  was triggered by. \nEach merge request can have multiple tasks related to it. So
+  step extracts all the messages of commits involved from the merge commit, \nand
+  it looks for task keys using a predefined format (for example, [ABCD-1234]). Then
+  using JIRA API, it updates custom fields of these tasks with current build number.
+  \n\nAdditionally step can keep ticket history from failed/aborted builds. It uses
+  Bitrise API to gather information about the commits \nfrom all aborted or failed
+  builds preceding the current one.\n\n### Useful links\n- [About this step](https://www.holdapp.com/blog/bitrise-tests-made-easier-with-jira-build-step)"
+website: https://github.com/Holdapp/bitrise-step-jira-build
+source_code_url: https://github.com/Holdapp/bitrise-step-jira-build
+support_url: https://github.com/Holdapp/bitrise-step-jira-build/issues
+published_at: 2021-02-02T17:08:57.274627+01:00
+source:
+  git: https://github.com/Holdapp/bitrise-step-jira-build.git
+  commit: 21663a6bb7ba0b8609778120ea1f12f89171f58b
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- notification
+toolkit:
+  go:
+    package_name: github.com/Holdapp/bitrise-step-jira-build
+deps:
+  brew:
+  - name: git
+  - name: libgit2
+  apt_get:
+  - name: git
+  - name: libgit2-dev
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: true
+inputs:
+- APP_VERSION: null
+  opts:
+    description: "If you develop an iOS project, you can extract the version number
+      from Info.plist of the main target. It’s called `CFBundleShortVersionString`.
+      \nAlternatively, you may use a custom step that can do it for you: [bitrise-step-xcode-build-version](https://github.com/nodes-ios/bitrise-step-xcode-build-version)"
+    is_required: true
+    summary: App version (for example 1.0.0)
+    title: App version
+- JIRA_HOST: null
+  opts:
+    is_required: true
+    summary: Your JIRA instance URL (e.g. https://company.atlassian.net)
+    title: JIRA host
+- JIRA_CUSTOM_FIELD_ID: null
+  opts:
+    description: "You can read how to find it [here](https://confluence.atlassian.com/jirakb/how-to-find-id-for-custom-field-s-744522503.html).
+      \nAnd if you don’t have a custom field yet, check out this [guide](https://confluence.atlassian.com/adminjiraserver/adding-a-custom-field-938847222.html)
+      that explains how to create it."
+    is_required: true
+    summary: Custom field id for the build number (integer)
+    title: JIRA Custom Field ID
+- JIRA_USERNAME: null
```

Comment:
Please only specify default values if they have definite values, and make sure they are strings. Check this please in all cases